Cargo.toml

```
@@ -0,0 +1,26 @@
+[package]
+name = "tree-sitter-sourcepawn"
+description = "sourcepawn grammar for the tree-sitter parsing library"
+version = "0.0.1"
+keywords = ["incremental", "parsing", "sourcepawn"]
+categories = ["parsing", "text-editors"]
+repository = "https://github.com/tree-sitter/tree-sitter-sourcepawn"
```

```suggestion
repository = "https://github.com/nilshelmig/tree-sitter-sourcepawn"
```
For what is this cargo file? Is it for compiling the webassembly binary?

---

IIRC the cargo file was generated by the tree-sitter CLI when I updated to the latest version.